platform-coop-toolkit/coop-map-directory-index
# feat: add community skills (resolves #85)

## Description

Adds field for community skills to user profiles (used for Community Builder role).

## Steps to test

1. Check out branch.
2. Run migrations for accounts.

**Expected behavior:** "Community skills" text field appears in account admin.

## Additional information

Not applicable.

## Related issues

- Blocker for #84.
- Resolves #85.
